go-chaos/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# syntax=docker/dockerfile:1.4
+FROM golang:alpine as builder
+
+WORKDIR /app 
+
+COPY --link go.mod go.sum .
+RUN go mod download
+
+COPY --link cmd/ ./cmd
```

Comment:
TIL: --link creates a symlink? Wasnt aware of this feature :+1:

Reply:
Not exactly but it allows to copy files independently from previous layers: https://docs.docker.com/engine/reference/builder/#copy---link

Reply:
Ok this is not what I expected but OK :D

Reply:
Yeah, I feel like a lot of the more recent docker features are really subtle and not easy to understand :grimacing: